exercises/reverse-string/.meta/src/reference/kotlin/ReverseString.kt

```diff
@@ -0,0 +1 @@
+fun reverse(input: String): String = input.reversed()
```

Comment:
Some maintainers dislike the use of built-in functions, though I personally have little problem with it seeing as learning library functions is an important part of learning a language IMO! 🙂

Reply:
I changed the implementation in a new commit.